packages/react-charting/src/components/HeatMapChart/HeatMapChartRTL.test.tsx

```diff
@@ -29,6 +30,105 @@ export const HeatMapData: IHeatMapChartProps['data'] = [
       },
     ],
   },
+  {
+    value: 200,
+    legend: 'Nasty',
+    data: [
+      {
+        x: datePoints[0],
+        y: stringPoints[1],
+        value: 50,
+        rectText: 50,
+        ratio: [50, 2391],
+        descriptionMessage: 'a good day to start with in Texas with best air quality',
+      },
+      {
+        x: datePoints[1],
+        y: stringPoints[0],
+        value: 25,
+        rectText: 25,
+        ratio: [25, 2479],
+        descriptionMessage: `Due to unexpected heavy rain, all the pollutants are washed
+      off and people of alaska are hoping for more of this days`,
+      },
+    ],
+  },
+];
+const HeatMapData2: IHeatMapChartProps['data'] = [
```

Comment:
Instead of 1, 2, 3 call these as string data, numeric data and date data.

Reply:
done